db/migrations/20240820140920-add-index-on-issue-resolution.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  async up (q) {
+    q.addIndex('remediation_issue_systems', ['resolved', 'remediation_issue_id']);
```

Comment:
Any reason to have the `resolved` first? `remediation_issue_id` has higher cardinality.

Reply:
Nope - that's just the way I typed it in my cold medicine induced haze...

Reply:
Fixed.